src/swsssdk/scripts/sonic-db-cli

```diff
@@ -36,3 +57,29 @@ else:
             print "\n".join(resp)
         else:
             print resp
+
+def main():
+    parser = argparse.ArgumentParser(description='SONiC DB CLI',
+                         formatter_class=argparse.RawTextHelpFormatter)
+    parser.add_argument('db_or_op', nargs='?', type=str, help='Database name Or Unary operation')
+    parser.add_argument('cmd', nargs='*', type=str, help='Command to execute in database')
```

Comment:
>add_argument [](start = 11, length = 12)

How to get usage help explicitly with command line argument?

Reply:
```
admin@ASW-7005:~$ sonic-db-cli -h
usage: sonic-db-cli [-h] db_or_op [cmd [cmd ...]]

SONiC DB CLI

positional arguments:
  db_or_op    Database name Or Unary operation(only PING supported)
  cmd         Command to execute in database

optional arguments:
  -h, --help  show this help message and exit

Example 1: sonic-db-cli CONFIG_DB keys *
Example 2: sonic-db-cli APPL_DB HGETALL VLAN_TABLE:Vlan10
Example 3: sonic-db-cli APPL_DB HGET VLAN_TABLE:Vlan10 mtu
Example 4: sonic-db-cli APPL_DB EVAL "return {KEYS[1],KEYS[2],ARGV[1],ARGV[2]}" 2 k1 k2 v1 v2
Example 5: sonic-db-cli PING

```